rootston/rootston.ini.example

```diff
@@ -30,6 +30,7 @@ meta-key = Logo
 
 # Keybindings
 # Maps key combinations with commands to execute
-# The special command "exit" stops the compositor
+# Use the prefix "exec" to execute a shell command
 [bindings]
-Logo+q = exit
+Logo+q = exit # Stops the compositor
+Logo+w = close # Closes the current view
```

Comment:
Just for fun let's match the default sway keybindings. mod+q closes the current view and mod+shift+e exits sway.